docs/connectors/getting-started.mdx

```diff
@@ -4,14 +4,14 @@ sidebar_position: 0
 sidebar_label: "Getting Started"
 ---
 
-A connector knows how to read/write records from/to a data source/destination
+A Conduit Connector knows how to read/write records from/to a data source/destination
 (e.g. a database).
 
 When thinking about connectors for Conduit, our goals were to:
-- provide a good development experience to connector developers,
-- ship Conduit with real built-in connectors (compiled into the Conduit binary),
-- to make it as easy as possible to write plugins in _any_ programming language,
-- the [Connector SDK](https://github.com/conduitio/conduit-connector-sdk) to be
+- Provide a good development experience to connector developers,
+- Ship Conduit with real built-in connectors (compiled into the Conduit binary),
+- To make it as easy as possible to write plugins in _any_ programming language,
+- The [Connector SDK](https://github.com/conduitio/conduit-connector-sdk) to be
```

Comment:
I think this was correct before, if you read it as a whole as a sentence that lists things. Now the sentences should be able to stand on their own and end with a period. I might be wrong though 🤷

Reply:
It's a matter of style I think. To me it read wrong before as lower case since that was inconsistent with how we were outlining things before. I just noticed we had commas at the end of each sentence so I guess this was written as separate sentences not a real list? 🤷